installer/PowerToysSetup/PowerToys.wxs

```diff
@@ -22,6 +22,7 @@
 
         <util:FileSearch Variable="HasDotnet3122" Path="[ProgramFiles64Folder]dotnet\shared\Microsoft.WindowsDesktop.App\3.1.22\System.Xaml.dll" Result="exists" />
         <util:FileSearch Variable="HasDotnet5013" Path="[ProgramFiles64Folder]dotnet\shared\Microsoft.WindowsDesktop.App\5.0.13\System.Xaml.dll" Result="exists" />
+        <util:RegistrySearch Variable="HasWebView2" Root="HKLM" Key="SOFTWARE\WOW6432Node\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" Format="raw" />
```

Comment:
@yuyoyuppe Please doublecheck this line. (ref https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#offline-deployment)